diktat-analysis.yml

```diff
@@ -339,5 +339,9 @@
   configuration: {}
 # Checks that there are abstract functions in abstract class
 - name: CLASS_SHOULD_NOT_BE_ABSTRACT
+  enabled: true
+  configuration: {}
+# In case of using "implicit backing property" scheme, the name of real and back property should be the same
```

Comment:
But what is described in our rule is pretty much the opposite of "implicit backing properties" - we are checking "explicit backing properties", like in [kotlin docs](https://kotlinlang.org/docs/reference/properties.html#backing-properties)